README.md

```diff
@@ -141,41 +141,37 @@ separately.
 - [Releases Page](RELEASES.md): Documentation for how to leverage our build artifacts.
 - [Roadmap for Support](ROADMAP.md): Documentation for our prioritized roadmap to support AMD GPUs.
 
-## Provisioning TheRock 🪨
+## Installing TheRock from artifacts 🪨
 
-In order to provision TheRock using either a developer/automated nightly release, a specific CI runner build or an already existing installation of TheRock, use the `build_tool/provision_machine.py` script.
+In order to install TheRock using either a developer/nightly release, a specific CI runner build or an already existing installation of TheRock, use the `build_tool/install_rocm_from_artifacts.py`. This script is used by CI to install TheRock to test machines and can be used locally to install TheRock from a specified source.
 
-Provisioning script setup:
+Script setup:
 
 - `python3 -m venv venv`
 - `source venv/bin/activate`
 - `pip install -r requirements.txt`
-- `python build_tools/provision_machine.py --help`
+- `python build_tools/install_rocm_from_artifacts.py --help`
 
 Examples:
 
-- Downloads the gfx94X S3 artifacts from GitHub CI workflow run 14474448215 (from [GitHub CI workflow run 14474448215](https://github.com/ROCm/TheRock/actions/runs/14474448215)) to the default output directory `therock-build`:
+- Downloads the gfx94X S3 all artifacts from GitHub CI workflow run 14474448215 (from [GitHub CI workflow run 14474448215](https://github.com/ROCm/TheRock/actions/runs/14474448215)) to the default output directory `therock-build`:
```

Comment:
This line is still verbose and repetitive. "GitHub CI workflow run 14474448215" appears twice and it should read "downloads all ... artifacts" instead of "downloads the ... all artifacts"